From c8da07269ba5502ad099d1071b78b3fe6b478c45 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Wed, 21 Mar 2018 14:29:45 +0800
Subject: [PATCH] phy: rockchip: inno-hdmi: no need to or efuse phy flag shift

The phy flag shift offset is defined in dts file, there is
no need to or the shift in the phy driver.

Change-Id: I13a33a536dabea68adf07a73cc2d13439719c589
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-inno-hdmi-phy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-inno-hdmi-phy.c b/drivers/phy/rockchip/phy-rockchip-inno-hdmi-phy.c
index 2ca9f81298e8..1c835426bb80 100644
--- a/drivers/phy/rockchip/phy-rockchip-inno-hdmi-phy.c
+++ b/drivers/phy/rockchip/phy-rockchip-inno-hdmi-phy.c
@@ -707,7 +707,7 @@ static void inno_hdmi_phy_rk3228_init(struct inno_hdmi_phy *inno)
 	efuse_buf = nvmem_cell_read(cell, &len);
 	nvmem_cell_put(cell);
 	if (len == 1)
-		inno->efuse_flag = (efuse_buf[0] & BIT(1)) ? true : false;
+		inno->efuse_flag = efuse_buf[0] ? true : false;
 	kfree(efuse_buf);
 }
 
-- 
2.35.3

